File: preview/hosted_numbers/hosted_number_order/fetch-default/fetch-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

hosted_number_order = @client.preview
  .hosted_numbers
  .hosted_number_orders('HRXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
  .fetch

puts hosted_number_order.friendly_name

File: preview/hosted_numbers/hosted_number_order/fetch-default/fetch-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Preview.HostedNumbers;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        // DANGER! This is insecure. See http://twil.io/secure
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var hostedNumberOrder = HostedNumberOrderResource.Fetch(
            pathSid: "HRXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
        );

        Console.WriteLine(hostedNumberOrder.FriendlyName);
    }
}
